src/config/tokens.ts

```diff
@@ -60,16 +69,24 @@ export const USDC: Token = Object.freeze({
   color: Color.usdcBlue,
   decimals: 18,
 })
+export const axlUSDC: Token = Object.freeze({
```

Comment:
I think there may be some confusion. The USDC token above _is_ the axelar USDC. If you merge latest into this branch you'll see that the name has changed.

Reply:
I see the name has changed in the most recent changes. 

To share a bit of my thinking process - while I was aware that the supported USDC variant is axlUSDC, I thought it would be a good idea to include it as a separate token in the list with the appropriate token ID and name.

I saw that the issue mentioned multiple USDC variants and came across several of them in the [Celo token list](https://github.com/celo-org/celo-token-list/blob/main/celo.tokenlist.json). So, I figured if Mento were to support additional USDC variants in the future, all we would need to do is add them to this list.

I hope this makes sense. Feel free to share your thoughts or correct me if I need to include something. Thanks for taking the time to review!